rancher/v2.0/en/quick-start-guide/index.md

```diff
@@ -4,91 +4,248 @@ layout: rancher-default-v2.0
+	> **Note:** Rancher v2.0 beta:
+	>
+	> -	Supports only the HTTPS protocol.
+	> -	Uses a self-signed certificate. Due to this signature, the browser prompts you to trust the certificate before login. Following GA, you'll be able to use your own certificate.
 
-### Creating Cloud Clusters
+### Create a Cluster
 
-In Rancher v2.0, you can create new Kubernetes clusters from hosted services like Google Container Engine (GKE).
+Begin using Rancher by creating your first cluster! A *cluster* is a group of physical (or virtual) computers that share resources to accomplish tasks as if they were a single system.
 
-1. Click on the **Add Cluster** button and in "Create a Cloud Cluster", click on **Select**.
+1.	Click **+ Add Cluster**.
 
-2.  Follow the instructions in the Rancher UI to create and add your cluster. The provisioning process might take a few minutes to complete. Once your cluster is ready, you can view its status on the Clusters page. Once your cluster is running, Rancher creates a `Default` project and a `default` namespace. Once the cluster is active, you can start adding pods into your namespace.
+	![add cluster](../../../../img/rancher/click-add-cluster.png)
 
-### Creating RKE Clusters
+	**Step Result:** The **Add Cluster** page opens.
 
-You can create a cluster using [RKE](https://github.com/rancher/rke), which will install Kubernetes on any nodes that you bring into your cluster. You can add node(s) from various cloud providers that Rancher v2.0 support and specify the role for each node for the Kubernetes cluster.
+2.	From the **Add Cluster** menu, choose a tile to create your first cluster (e.g. **Digital Ocean**, **Azure Container Service**, etc.).
 
-> **Note:** Currently in the tech preview, only Digital Ocean and AWS nodes are supported, but support for all the cloud providers and adding custom nodes will be added.
+	> **Note:**
+	>
+	> -	For Rancher v2.0 beta, Amazon EKS is not supported. This option will be available after GA.
+	> -	For this tutorial, the Import option is out of scope. For now, create a cluster using one of the other options. We'll address Import later.
 
-<!--
-If you're adding a custom host, note these requirements:
+3.	Enter a **Cluster Name**. No spaces allowed.
+
+	> **Tip:** Skip adding **Member Roles** for now. This option isn't essential for your first cluster.
+	>
+	> ![skip member roles](../../../../img/rancher/skip-member-roles.png)
+
+4.	Select **Cluster Options**.
+
+	Use these options to choose things like the version of Kubernetes that's installed in your cluster, along with other Kubernetes options such as pod security policies. Some services have more options than others. If you're unsure of what to choose, use the default options.
+
+	**Using Google Container Engine?** Skip to the next step. GCE doesn't have any cluster options.
```

Comment:
Google Container Engine's acronym is GKE. Also, GKE has cluster options after you enter your service account info. 

<img width="1142" alt="screen shot 2018-04-01 at 9 09 20 pm" src="https://user-images.githubusercontent.com/11410997/38182859-0f82bbb0-35f1-11e8-9868-e3bc3d84d292.png">